content/2015.07.cliquet.rationale-en.rst

```diff
@@ -0,0 +1,494 @@
+Why did we make Cliquet?
```

Comment:
How about going in a different direction? "Making the case for Cliquet", for example? It doesn't necessarily need to be a literal translation...